### ISSUE TITLE
As an (S)RA(A) i want to see the configured vetting methods

### ISSUE THREAD

**Issue by phavekes** [OpenConext/Stepup-Project#366]
[This issue is imported from pivotal - Originaly created at Sep 26, 2022 by Peter Havekes](https://www.pivotaltracker.com/story/show/183376863)

Shor the status of the self-vetting and self-asserted configuration for an institution.

**Comment by phavekes** [comment]
(Peter Havekes - Sep 26, 2022)

**Comment by phavekes** [comment]
@phavekes I suggested the following EN translations. The Dutch ones I picked from your wireframe/screenshot. Feel free to suggest translation suggestions.

Vertaalkey, EN, NL
`ra.institution_configuration.allow`: Allowed, Toegestaan
`ra.institution_configuration.disallow`: Not allowed, Niet toegestaan
`ra.institution_configuration.allow_self_vet`: Token activation using an activated token, Bestaand token gebruiken om te activeren
`ra.institution_configuration.allow_self_assertedTokens`: Activate a token without the service desk or an activated token, Token activeren zonder servicedesk of bestaand token (Michiel Kodde - Oct 24, 2022)
